### PR TITLE
Include Generated Header/Implementation Files

### PR DESCRIPTION
**Pull request by dmccoystephenson**
## Problem
We recently removed the J2735 ASN files as we are not permitted to redistribute them. This has caused the travis build to start failing as it is unable to generate the necessary C files at build time.

## Solution
The generated header/implementation files for 2016 & 2020 have been included in the project in compressed form.

The generation part of the `doIt.sh` script has been moved to the `generate-files.sh` script. The `doIt.sh` script is now responsible for retrieving the proper files & extracting them into the `asn1c_combined` directory.

## Testing
- The project is able to built & run with these changes using Docker.
- The unit tests pass.
- Running a message through the program in AEM mode has been verified to be processed successfully.

### CODE REVIEW

**Review by drewjj — APPROVED** [review]
This looks great. It tested fine locally.

**Review by mwodahl — APPROVED** [review]
This looks good! It ran correctly locally

File: Dockerfile [hunk 1]
```diff
@@ -29,12 +29,19 @@ RUN cd /asn1_codec/pugixml && mkdir -p build && cd build && cmake .. && make &&
...
 
-# Generate ASN.1 API.
+# Make generated files available to the build & compile example
 RUN export LD_LIBRARY_PATH=/usr/local/lib
 ADD ./asn1c_combined /asn1_codec/asn1c_combined
 ADD ./scms-asn1 /asn1_codec/scms-asn1
 RUN cd /asn1_codec/asn1c_combined && bash doIt.sh
 
+# Remove any lingering .asn files
+RUN rm -rf /asn1c_codec/asn1c_combined/j2735-asn-files
```
Comment (payneBrandon) [inline Dockerfile:39]: These should already be removed right? Do we need this?

File: asn1c_combined/generate-files.sh [hunk 2]
```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+# This script generates the J2735 files for a given year.
```
Comment (payneBrandon) [inline asn1c_combined/generate-files.sh:3]: This script relies on the asn files being available in the solution, right? Should we add a comment in here to that effect?

Comment (payneBrandon) [reply asn1c_combined/generate-files.sh:3]: Looks like that's well-covered in the README, disregard

**Review by payneBrandon — CHANGES_REQUESTED** [review]
Just the one question on necessity of removing files

File: Dockerfile (hunk 1 above)
Comment (dmccoystephenson) [reply Dockerfile:39]: If the user has any ASN files in the `asn1c_combined/j2735-asn-files` directory at build time, they will be added to the image on line 34:
> ADD ./asn1c_combined /asn1_codec/asn1c_combined

In the event that the user has just regenerated the implementation/header files using the `generate-files.sh` script files, this ensures that the ASN files the script references in the 'j2735-asn-files' directory don't make it into the image.

**Review by payneBrandon — APPROVED** [review]
lgtm!

**Pull request merged**